nsv13/code/modules/cargo/packs.dm

```diff
@@ -457,3 +457,40 @@
 	cost = 750
 	contains = list(/obj/item/vending_refill/wardrobe/muni_wardrobe)
 	crate_name = "munidrobe supply crate"
+
+datum/supply_pack/medical/chemical_supply
+	name = "Chemical Supply Crate"
+	desc = "This crate contains 10 bottles of chemical elements"
+	cost = 3000
```

Comment:
```suggestion
	cost = 1000
```
3000 is FAR too much for a random chance of getting 30u of the chems you need,.

Reply:
Use the medical budget.

Reply:
the issue isn't sourcing funds, it's that this is obscenely overpriced, especially when compared to its fellow medical crates, ESPECIALLY considering that this is a random chance crate in which you aren't even guaranteed to get the stuff you need.

Singlepacks for medkits cost 330 credits, the variety medical crate costs 2000, the vendor refills cost 2000, the synthflesh resupply crate costs the same, and there's even a chemical starter crate for 1700 which has 13 different chemical bottles __guaranteed__! Admittedly, the variety is thinner than this, but it has everything you'd need to mix basic medicines.

Reply:
Adjust down a bit.